iothub/device/src/Transport/Mqtt/MqttTransportHandler.cs

```diff
@@ -910,7 +910,7 @@ public override async Task SendTwinPatchAsync(TwinCollection reportedProperties,
 
                 var clientTlsSettings = new ClientTlsSettings(
                      protocols,
-                     certs.Any(),
+                     settings.CertificateRevocationCheck,
```

Comment:
Should be certs.Any() && settings.CertificateRevocationCheck, because you don't want to check if there aren't certs.